src/multipage.jl

```diff
@@ -0,0 +1,185 @@
+ansi_moveup(n::Int) = string("\e[", n, "A")
+ansi_movecol1 = "\e[1G"
+ansi_cleartoend = "\e[0J"
+ansi_enablecursor = "\e[?25h"
+ansi_disablecursor = "\e[?25l"
+
+"""
+    play(io::IO, arr::T, dim::Int; kwargs...)
+    play(io::IO, framestack::Vector{T}; kwargs...) where {T<:AbstractArray}
+
+Play a video of a framestack of image arrays, or 3D array along dimension `dim`.
+
+Control keys:
+- `p` or `space-bar`: pause/resume
+- `f`, `←`(left arrow), or `↑`(up arrow): step backward
+- `b`, `→`(right arrow), or `↓`(down arrow): step forward
+- `ctrl-c` or `q`: exit
+
+kwargs:
+- `fps::Real=30`
+- `maxsize::Tuple = displaysize(io)`
+"""
+function play(io::IO, arr::T, dim::Int; fps::Real=30, maxsize::Tuple = displaysize(io), paused = false) where {T<:AbstractArray}
+    @assert dim <= ndims(arr) "Requested dimension $dim, but source array only has $(ndims(arr)) dimensions"
+    @assert ndims(arr) <= 3 "Source array dimensions cannot exceed 3"
+    firstframe = T <: Vector ? first(selectdim(arr, dim, 1)) : selectdim(arr, dim, 1)
```

Comment:
It's a little bit dirty here to put the framestack support here. Indeed, a better strategy is to put a lazy-view to convert `Vector{<:AbstractArray{T, 2}}` to `Array{T, 3}`.

```diff
- play(io::IO, framestack::Vector{T}; kwargs...) where {T<:AbstractArray} = play(io, framestack, 1; kwargs...)
+ play(io::IO, framestack::Vector{T}; kwargs...) where {T<:AbstractArray} = play(io, 3dview(framestack), 3; kwargs...)
```

Do you know if there's such a `3dview` tool in Julia? Or I can make one for this.